src/components/NavBar/NavBar.jsx

```diff
@@ -0,0 +1,179 @@
+              <Typography variant="h4" noWrap component="div" mr="10px">
+                PASS
+              </Typography>
+              <>
+                <Typography
+                  variant="h5"
+                  noWrap
+                  component="div"
+                  sx={{
+                    display: 'flex',
+                    flexDirection: 'row',
+                    alignItems: 'center',
+                    '& > *': {
+                      m: 1
+                    }
+                  }}
+                >
+                  <Link to="/PASS/home/">Home</Link>
+                  <Link to="/PASS/forms/">Forms</Link>
+                </Typography>
+
+                <Box sx={{ flexGrow: 1 }} />
+                <Box sx={{ display: { xs: 'none', md: 'flex' } }}>
+                  <IconButton
+                    size="large"
+                    aria-label="show new notifications"
+                    color="inherit"
+                    onClick={handleNotificationsMenu}
+                  >
+                    <Badge color="error">
+                      <NotificationsIcon />
+                    </Badge>
+                  </IconButton>
+                  <Menu
+                    id="menu-appbar-notifications"
+                    anchorEl={anchorElNotifications}
+                    anchorOrigin={{
+                      vertical: 'top',
+                      horizontal: 'right'
+                    }}
+                    keepMounted
+                    transformOrigin={{
+                      vertical: 'top',
+                      horizontal: 'right'
+                    }}
+                    open={Boolean(anchorElNotifications)}
+                    onClose={handleNotificationsClose}
+                    sx={{ marginTop: '40px' }}
+                  >
+                    <MenuItem onClick={handleNotificationsClose}>
+                      <p>Notification 1</p>
+                    </MenuItem>
+                    <MenuItem onClick={handleNotificationsClose}>
+                      <p>Notification 2</p>
+                    </MenuItem>
+                  </Menu>
+                  <IconButton
+                    size="large"
+                    edge="end"
+                    aria-label="account of current user"
+                    aria-controls={menuId}
+                    aria-haspopup="true"
+                    onClick={handleOpenMenu}
+                    color="inherit"
+                  >
+                    <AccountCircle />
+                  </IconButton>
+                </Box>
+                <Box sx={{ display: { xs: 'flex', md: 'none' } }}>
+                  <IconButton
+                    size="large"
+                    aria-label="show more"
+                    aria-controls={mobileMenuId}
+                    aria-haspopup="true"
+                    onClick={handleOpenMobileMenu}
+                    color="inherit"
+                  >
+                    <MoreIcon />
```

Comment:
In Line 141, I think we should use the hamburger menu icon rather than the vertical more icon.